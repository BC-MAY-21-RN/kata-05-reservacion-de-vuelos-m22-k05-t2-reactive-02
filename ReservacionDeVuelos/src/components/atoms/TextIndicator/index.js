import React from 'react';
import {View, Text} from 'react-native';
import styles from './style';

export default function TextIndicator({text}) {
  return (
    <View>
      <Text style={styles.title}>{text}</Text>
    </View>
  );
}
