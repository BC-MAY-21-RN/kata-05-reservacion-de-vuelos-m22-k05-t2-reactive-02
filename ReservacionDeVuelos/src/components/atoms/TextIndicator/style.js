import {StyleSheet} from 'react-native';

const styles = StyleSheet.create({
  title: {
    fontSize: 32,
    marginTop: 220,
    fontWeight: 'bold',
    color: 'black',
    marginHorizontal: 30,
  },
});

export default styles;
