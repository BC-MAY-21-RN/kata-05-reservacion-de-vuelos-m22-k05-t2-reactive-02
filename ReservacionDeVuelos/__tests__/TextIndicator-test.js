import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import TextIndicator from '../src/components/atoms/TextIndicator';

jest.mock('react-native/Libraries/EventEmitter/NativeEventEmitter');

describe('Tests TextIndicator component', () => {
  it('TextIndicator snapshot', () => {
    const renderTextIndicator = renderer
      .create(<TextIndicator text={''} style={{}} />)
      .toJSON();
    expect(renderTextIndicator).toMatchSnapshot();
  });
});
