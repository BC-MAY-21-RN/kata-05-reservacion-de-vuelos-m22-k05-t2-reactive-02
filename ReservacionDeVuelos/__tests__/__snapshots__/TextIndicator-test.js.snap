// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests TextIndicator component TextIndicator snapshot 1`] = `
<View
  style={Object {}}
>
  <Text
    style={
      Object {
        "color": "black",
        "fontSize": 32,
        "fontWeight": "bold",
        "marginHorizontal": 30,
      }
    }
  >
    
  </Text>
</View>
`;
